yt_dlp/extractor/piaulizaportal.py

```diff
@@ -49,18 +49,22 @@ class PIAULIZAPortalAPIIE(InfoExtractor):
 
     def _real_extract(self, url):
         url, smuggled_data = unsmuggle_url(url, {})
-        video_id = smuggled_data.get('video_id') or self._search_regex(r'&name=([^&]+)', self._match_id(url), 'video id')
+        tmp_video_id = smuggled_data.get('video_id') or self._search_regex(r'&name=([^&]+)', self._match_id(url), 'video id', default='unknown')
         player_data = self._download_webpage(
             url,
-            video_id,
+            tmp_video_id,
             headers={'Referer': smuggled_data.get('referer') or 'https://player-api.p.uliza.jp/'},
             note='Fetching player data', errnote='Unable to fetch player data',
         )
 
+        m3u8_url = self._search_regex(
+            r'["\'](https://vms-api\.p\.uliza\.jp/v1/prog-index\.m3u8[^"\']+)', player_data,
+            'm3u8 url', default=None)
+
+        video_id = smuggled_data.get('video_id') or self._search_regex(r'&?ss=([\da-f]{8}-(?:[\da-f]{4}-){3}[\da-f]{12})&?', m3u8_url, 'video id', default=tmp_video_id)
```

Comment:
The m3u8 url has the query like `ss=uuid`, so I extract the uuid from the query as the video id.
But unfortunately, the video title still can't be got from the raw m3u8 url.